### PR TITLE
add canary name and branch pass through

### PR DESCRIPTION
**Pull request by agentiqd**

### CODE REVIEW

File: .github/workflows/docker-build-deploy.yaml [hunk 1]
```diff
@@ -290,13 +290,13 @@ jobs:
...
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      SERVICE_NAME: "dashboard-v1"
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       TARGET_NODE_VERSION: 18.8
       TARGET_PYTHON_VERSION: 3.9
       CANARY_CONDITIONAL: "${{ needs.check-canary.outputs.canary }}"
     steps:
-      - name: 🦜 Pass
+      - if: ${{ env.CANARY_CONDITIONAL == '' }}
```
Comment (agentiqd) [inline .github/workflows/docker-build-deploy.yaml:298]: @BlaiseAgentIQ Is this acceptable for the workflow? I noticed it always does "Passes"

File: .github/workflows/docker-build-deploy.yaml [hunk 2]
```diff
@@ -290,13 +290,13 @@ jobs:
       AWS_DEFAULT_REGION: "us-east-1"
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      SERVICE_NAME: "dashboard-v1"
```
Comment (agentiqd) [inline .github/workflows/docker-build-deploy.yaml:293]: Gets overwritten later but makes me wonder where you got this snipper 😉

File: .github/workflows/docker-build-deploy.yaml (hunk 1 above)
Comment (BlaiseAgentIQ) [reply .github/workflows/docker-build-deploy.yaml:298]: That should be fine I think. I just needed a dummy that'll always run so the whole job doesn't get skipped and cause havoc down stream. But skipping when others run should work fine.

File: .github/workflows/slack-update.yaml [hunk 3]
```diff
@@ -177,10 +182,10 @@ jobs:
         uses: trstringer/manual-approval@v1.9.0
         with:
           secret: ${{ secrets.GITHUB_TOKEN }}
-          approvers: BlaiseAgentIQ,agentiqd
+          approvers: BlaiseAgentIQ,agentiqd,akspha,ryaniq
           minimum-approvals: 1
-          issue-title: "Deploying a test"
```
Comment (BlaiseAgentIQ) [inline .github/workflows/slack-update.yaml:182]: Oh that's embarrassing. I forgot to go back in and clean that up. Thank you for beautifying this! 🙏

File: .github/workflows/slack-update.yaml [hunk 4]
```diff
@@ -177,13 +182,13 @@ jobs:
...
-          issue-body: "Please approve or deny the test deployment."
+          issue-title: "Canary deployed ${{ inputs.deploy_branch }} to ${{ inputs.canary_target }}"
+          issue-body: "Please test and ensure environment integrity. Once tests are complete approve to proceed or deny to halt the deployment."
           exclude-workflow-initiator-as-approver: false
-          additional-approved-words: 'good,lg,✅,✔️,☑️,✓,✔,👍'
-          additional-denied-words: 'bad,stop,error,failed,fail,broken,❌,🛑,👎,❎'
```
Comment (agentiqd) [inline .github/workflows/slack-update.yaml:186]: adds "proceed" to approved and removes the green X emoji. I think using green to stop a deploy is misleading.

**Review by BlaiseAgentIQ — APPROVED** [review]
Looks great!!

**Pull request merged**